fr.yml

```diff
@@ -0,0 +1,202 @@
+---
```

Comment:
Please delete this file, the actual french locale is already in `config/locales/fr.yml`